React.md

```diff
@@ -2,6 +2,8 @@
 
 The biggest different is on reactive handling. In React, every change is handled by the reconciler that runs periodically. Whereas in Solid, only reactive changes will lead to re-rendering, which is great for performance and allows a good level of control - however, using `Array.map` will only ever run initially. If the contents of the array changed, you'd either need `mapArray` or `For`. 
 
+Also, you can't destructure `props` in Solid because it will lose reactivity. You can see it [this codesandbox](https://codesandbox.io/s/trusting-jasper-ltzxi?file=/src/main.tsx).
```

Comment:
```suggestion
Also, you can't destructure `props` in Solid because it will lose its reactivity. You can see it in [this codesandbox](https://codesandbox.io/s/trusting-jasper-ltzxi?file=/src/main.tsx).
```